conveyal/analysis-ui
# Add size limit to OSM and GTFS upload

- [x] Add limits to all other file uploads.

Add a 500MB limit to individual OSM and GTFS upload fields and a 1GB total limit. Show an error message when a user selects a file or files that exceed those limits. 

closes #1478 and #1313

## How to test

1. Select an OSM file that's more than 500MB and ensure that the error message shows and you cannot click the upload button.
2. Repeat for the GTFS field.
3. Select multiple files that are smaller than 500MB but exceed 1GB and ensure the error is shown.
